.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
     - rm -rf ${CONDA_INSTALL_LOCN}/pkgs && ln -s ${HOME}/cache/pkgs ${CONDA_INSTALL_LOCN}/pkgs
 
     # Now do the things we need to do to install it.
-    - conda install --file requirements.txt coverage coveralls mock pytest pytest-cov ${CONDA_PKGS} --yes --quiet ${CHANNEL}
+    - conda install --file requirements.txt coverage coveralls mock pytest pytest-cov shellcheck ${CONDA_PKGS} --yes --quiet ${CHANNEL}
```

Comment:
Shouldn't this be in `requirements.txt`?

Reply:
Well, isn't the `requirements.txt` (like `install_requires` in `setup.py`) only for python packages, whereas `shellcheck` is a cli tool and a conda but not a python package?

conda-build handles `ripgrep` in a similar way: Not part of the `install_requires`, but installed via conda during CI and part of the run dependencies in the meta.yml.

Reply:
That is also the reason why ripgrep/shellcheck are use in a way that they are no hard dependencies, meaning conda-build and recipe-lint also both work if the cli commands are not available.

Reply:
>  isn't the requirements.txt only for python packages

No, I don't think it is.  At least not generally for conda.

Reply:
Ok. Will change it then. (Some packages have the file conda & pip install compatible which is anyway not the case here)